src/main/java/net/nuagenetworks/bambou/RestObject.java

```diff
@@ -334,11 +334,12 @@ public void assign(RestSession<?> session, List<? extends RestObject> childRestO
 
     @Override
     public void assign(RestSession<?> session, List<? extends RestObject> childRestObjs, boolean commit) throws RestException {
-        // Make sure there are child objects passed in
-        if (childRestObjs.isEmpty()) {
-            throw new RestException("No child objects specified");
-        }
-
+    	
+    	// Make sure the child objects passed in is not null     
+    	if (childRestObjs == null) {	
```

Comment:
If childRestObjs size is 0, there will be an exception thrown on line 349 where the first entry in the childRestObjs list is used to get the type of object.